db/dbtypes/types.go

```diff
@@ -9,10 +9,43 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/decred/dcrdata/v3/db/dbtypes/internal"
 )
 
+// TimeDef is time.Time wrapper that formats time by default as a string without
+// a timezone. The time Stringer interface formats the time into a string
+// with a timezone.
+type TimeDef struct {
+	T time.Time
+}
+
+func (t TimeDef) String() string {
+	return t.T.Format("2006-01-02 15:04:05")
+}
+
+// MarshalJSON is set as the default marshalling function for TimeDef struct.
+func (t *TimeDef) MarshalJSON() ([]byte, error) {
+	return json.Marshal(t.String())
+}
+
+// TimeAPI is a fall back dbtypes.TimeDef wrapper that allows API endpoint that
+// previously accepted timestamp instead of a formatted string time to continue
+// working.
+type TimeAPI struct {
```

Comment:
Having both struct share a name even though they will located in different packages is quite confusing. I propose we stick with `dbtypes.TimeDef` and `apitypes.TimeAPI` just to help differentiate the structs on inspection.

Reply:
I would also not advocate we name either struct as `Time` since the default time field on several structs is `Time`.
```golang
Time  []Time `json:"time,omitempty"`
```
This looks not right to me.

Reply:
No problem keeping the names.  Personally I find the import prefixes enough to disambiguate, but it's a minor style issue. I'm also not personally bothered by `Time []Time`, but I understand it can be hard to swallow.
PR looks good.